cobald_tests/daemon/config/test_python.py

```diff
@@ -0,0 +1,29 @@
+import tempfile
+import os
+
+from cobald.daemon.config.python import load_configuration
+
+
+def module_content(identifier):
+    return f"""\
+identifier = {identifier}
+"""
+
+
+def test_load_pyconfig():
```

Comment:
You might consider adding a test for wrong file extensions to cover all branches.

Reply:
Added a test that covers what I think are common issues:

* Python file extension that is not a module (``.pyi``)
* Config file extension that is not Python (``.yml``)
* Missing file extension